client/components/data/query-site-purchases/index.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
```

Comment:
You need to provide `React` to make sure JSX transpilation works properly. It might work without, but I already learned myself that it's pretty random.